.github/workflows/clang-tidy.yml

```diff
@@ -124,7 +124,7 @@ jobs:
           build_dir: build-clang-tidy
           config_file: ".clang-tidy"
           split_workflow: true
-          exclude: "lib/*"
+          exclude: "lib/*,auxiliary/*"
```

Comment:
This should be changed before we merge it in, as this is converted to a submodule & moved